.github/workflows/empty-worksapce-smoke-test-on-minikube-ubi9.yaml

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: Empty workspace smoke test on udi9
+on:
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - .devfile.yaml
+      - LICENSE
+      - '.rebase/*'
+      - 'base/ubi9/**'
```

Comment:
base/ubi8/** and /universal/ubi8/** should be ignored and .rebase/* can be removed (it doesn't exist).
Also in `empty-worksapce-smoke-test-on-minikube-ubi8.yaml` you can extend `paths-ignore` by adding 'universal/ubi9/**'